### PR TITLE
fix the wrong message printed about how to enable logging for the cluster...

### PR DESCRIPTION
**Pull request by kreuzwerker-patrick**
… - type was not specified, therefore the command didn't work

### Description

I have fixed a wrong display message when creating a cluster with no logging.
The message was missing a flag for the command - the current message doesn't work...

### Checklist
- [x] Added tests that cover your change (if possible)
- [x] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [x] Manually tested
- [ ] Added labels for change area (e.g. `area/nodegroup`), target version (e.g. `version/0.12.0`) and kind (e.g. `kind/improvement`)
(I think I don't have permission to add labels... :-()
- [x] Make sure the title of the PR is a good description that can go into the release notes

### CODE REVIEW

File: pkg/eks/tasks.go [hunk 1]
```diff
@@ -117,7 +117,7 @@ func (c *ClusterProvider) CreateExtraClusterConfigTasks(cfg *api.ClusterConfig,
 	}
 	if !cfg.HasClusterCloudWatchLogging() {
 		logger.Info("CloudWatch logging will not be enabled for cluster %q in %q", cfg.Metadata.Name, cfg.Metadata.Region)
-		logger.Info("you can enable it with 'eksctl utils update-cluster-logging --region=%s --cluster=%s'", cfg.Metadata.Region, cfg.Metadata.Name)
+		logger.Info("you can enable it with 'eksctl utils update-cluster-logging --enable-types=all --region=%s --cluster=%s'", cfg.Metadata.Region, cfg.Metadata.Name)
```
Comment (michaelbeaumont) [inline pkg/eks/tasks.go:120]: I think the idea might be to force the user to see the error message and choose for themselves what logging should be enabled. What do you think?

Comment (kreuzwerker-patrick) [reply pkg/eks/tasks.go:120]: OK, then I would personally prefer to add "--enable-types={SPECIFY-LOG-TYPES}" or something like that....
Cause the error message after executing the command does not immediately imply the correct usage (without consulting the help).
But indeed it is also one of the many ways that lead to Rome ... ;-)

**Comment by martina-if** [comment]
Hi @kreuzwerker-patrick thank you for this PR, do you think you'll have time to make the small changes soon?

**Comment by kreuzwerker-patrick** [comment]
> Hi patrick thank you for this PR, do you think you'll have time to make the small changes soon?

done...

File: userdocs/src/usage/fargate-support.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ $ eksctl create cluster --fargate
 [ℹ]  will create 2 separate CloudFormation stacks for cluster itself and the initial nodegroup
 [ℹ]  if you encounter any issues, check CloudFormation console or try 'eksctl utils describe-stacks --region=ap-northeast-1 --cluster=ridiculous-painting-1574859263'
 [ℹ]  CloudWatch logging will not be enabled for cluster "ridiculous-painting-1574859263" in "ap-northeast-1"
-[ℹ]  you can enable it with 'eksctl utils update-cluster-logging --region=ap-northeast-1 --cluster=ridiculous-painting-1574859263'
+[ℹ]  you can enable it with 'eksctl utils update-cluster-logging --enable-types=all --region=ap-northeast-1 --cluster=ridiculous-painting-1574859263'
```
Comment (michaelbeaumont) [inline userdocs/src/usage/fargate-support.md:27]: this should probably also show the actual message, also below

**Review by michaelbeaumont — CHANGES_REQUESTED** [review]

File: userdocs/src/usage/fargate-support.md (hunk 2 above)
Comment (kreuzwerker-patrick) [reply userdocs/src/usage/fargate-support.md:27]: right, done...

Comment (michaelbeaumont) [reply userdocs/src/usage/fargate-support.md:27]: still missing one..

Comment (kreuzwerker-patrick) [reply userdocs/src/usage/fargate-support.md]: after grepping through all the files, I have not replaced all relevant messages accordingly.

**Review by michaelbeaumont — APPROVED** [review]

**Comment by michaelbeaumont** [comment]
Thanks @kreuzwerker-patrick

**Pull request merged**